src/fidesops/analytics.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import os
+from platform import system
+from typing import Dict
+
+from fideslog.sdk.python.client import AnalyticsClient
+from fideslog.sdk.python.event import AnalyticsEvent
+from fideslog.sdk.python.exceptions import AnalyticsError
+from fideslog.sdk.python.utils import FIDESOPS, generate_client_id
+
+from fidesops.core.config import config, update_config_file
+
+from ._version import get_versions
+
+logger = logging.getLogger(__name__)
+
+
+def get_version() -> str:
+    """Version of Fidesops"""
+    return get_versions()["version"]
+
+
+def in_developer_mode() -> bool:
+    """True if running in dev mode"""
+    return config.dev_mode
+
+
+def in_docker_container() -> bool:
+    """`True` if the command was submitted within a Docker container. Default: `False`."""
+    return True
```

Comment:
I wouldn't assume this will always be `True`. In fidesctl, we [create an ENV variable when running in docker](https://github.com/ethyca/fides/blob/a1e08d7b2565e01a82f70ad7625179620db54c74/Dockerfile#L90-L91). Then, we [read the variable at run time](https://github.com/ethyca/fides/blob/4e6b6591406a842ca5376109bf10b862d69f147c/src/fidesctl/cli/utils.py#L143) to make this determination. I'd recommend doing the same here.

Reply:
good call, I missed this.